Hmac.js

```diff
@@ -66,12 +82,23 @@ export class Hmac {
    * @param {Uint8Array} options.data - The data to sign as a Uint8Array.
    * @param {string} options.signature - The base64url-encoded signature
    *   to verify.
+   * @param {boolean} options.useCache - Enable the use of a cache.
    *
    * @returns {Promise<boolean>} `true` if verified, `false` if not.
    */
-  async verify({data, signature}) {
+  async verify({data, signature, useCache = true}) {
+    if(useCache) {
+      const sig = this.cache.get(base64url.encode(data));
```

Comment:
This should be named `verified` or similar.  It's a boolean.

Reply:
Yeah, these verify changes don't seem right. Instead, if the signature is in the cache, it would hit the cache and do a timing-safe compare between the given signature and the cached one -- and return a boolean for that.

Reply:
@dlongley Is a timing-safe compare important in the updated code?